modules/machines-edge-machine-pool.adoc

```diff
@@ -0,0 +1,103 @@
+
+:_content-type: CONCEPT
+[id="machines-edge-machine-pool_{context}"]
+= The edge machine pool
+
+{product-title} 4.12 introduced a new compute pool, _edge_, that is designed for use in remote zones. The edge compute pool configuration is common between AWS Local Zone locations. However, due to the image type and size limitations on Local Zone resources, the default instance type that is created can vary from the traditional worker pool.
+
+The default elastic block store (EBS) for Local Zone locations is `gp2`, which differs from the default worker pool. The preferred list of instance types for the edge compute pool follows the same order of worker pools. Depending on the availability of the location, the system chooses one of those instances. The preferred list of instance types for the edge compute pool includes:
+
+* `m6i.xlarge`
+* `m5.xlarge`
+* `c5d.2xlarge`
+
+The edge compute pool creates new labels that developers can use to deploy their applications onto those locations. The new labels are:
+
+* `node-role.kubernetes.io/edge=''`
+* `machine.openshift.io/zone-type=local-zone`
+* `machine.openshift.io/zone-group=<$ZONE_GROUP_NAME>`
+
+
+By default, the system creates the edge compute pool only if users add AWS Local Zone subnet IDs to the list `platform.aws.subnets`.
```

Comment:
This will be changed on 4.14 and we will support to create subnets when the zone will be added on the edge pool.

I would suggest reviewing that phrase to something like: "By default, the installer creates the edge compute pool manifests when installing in existing VPC, only when the subnet ID is added to the list `platform.aws.subnets`.

- Phase-1 (this, 4.13): create edge manifests when installing in existing VPC
- Phase-2 (plan, 4.14): create the zone name for Local Zones is specified in the edge compute pool

```suggestion
By default, the installer creates the edge compute pool manifests only if users add AWS Local Zone subnet IDs to the list `platform.aws.subnets` on install-config.yaml.
```

Reply:
> the installer creates the edge compute pool manifests 

installer will create **the manifests**, not the pool itself. I think stating this could provide more understanding what the installer is doing. wdyt?

Reply:
Fair. 

```suggestion
By default, the system creates the edge compute pool manifest only if users add AWS Local Zone subnet IDs to the list `platform.aws.subnets`.
```